File: JavaScript/1589.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var num = parseInt(lines.shift()); 

  for (var i = 0; i < num ; i++) {
    var raios = lines.shift().split(" ");
    var raio1 = parseInt(raios[0]);
    var raio2 = parseInt(raios[1]);
    console.log(raio1 + raio2)
  }

File: Java/1589.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
  public static void main(String[] args) throws IOException {
    Scanner leitor = new Scanner(System.in);
    
    int raio1, raio2;

    int num = leitor.nextInt();

    for(int i = 1; i <= num; i++){
          raio1 = leitor.nextInt();
        	raio2 = leitor.nextInt();
        	System.out.println(raio1 + raio2);
    }
  }
}
